pkmntypes.py

```diff
@@ -22,3 +22,18 @@ def __init__(self, **kwargs):
 
 class Pokemon():
 	pass
+
+class BattleContext():
+	"""This is a class that describes the point of view of a given Pokemon on the battlefield. It provides
+	enough information for a user to make an informed decision about what turn to make next.   
+	"""
+	# will add make the docstrings comply with the sphinx format once this class is looking good
+
+	def __init__(self, **kwargs):
+
+		self.battle = data['Battle']
+		self.pokemon = data['Pokemon']
+		self.team = data['Team']
+		self.targets = data['Targets']
+		self.allies = data['Allies']
+		self.opponents = data['Opponents']
```

Comment:
you also need to change these as per my review comment

Reply:
Yup, I already did, but I'm gonna push up those changes with the docstring edit. Unless you want me to commit them separately, lmk

Reply:
No just push them up together